khtools/jaccard_utils.py

```diff
@@ -1,24 +1,24 @@
-
-from joblib import Parallel, delayed
-import itertools
-
-
 from .idf import filter_idf
 
+
 def jaccard_sigs(i, j, siglist):
     return siglist[i].jaccard(siglist[j])
 
 
-def jaccard_sigs_idf(i, j, siglist, idf, mean_idf_per_cell, ):
-    i_hashes = filter_idf(siglist[i].get_mins(),
-                          idf, mean_idf_per_cell)
-    j_hashes = filter_idf(siglist[j].get_mins(),
-                          idf, mean_idf_per_cell)
+def jaccard_sigs_idf(
+        i,
+        j,
+        siglist,
+        idf,
+        mean_idf_per_cell,
+):
+    i_hashes = filter_idf(siglist[i].get_mins(), idf, mean_idf_per_cell)
+    j_hashes = filter_idf(siglist[j].get_mins(), idf, mean_idf_per_cell)
     return jaccard(i_hashes, j_hashes)
 
 
 def jaccard(sample1, sample2):
```

Comment:
this exists in a different module again! remove the other module jaccard definition as it can be imported from here instead